unattended_installer/common_functions/common.sh

```diff
@@ -134,8 +134,29 @@ function common_checkWazuhConfigYaml() {
 
 }
 
+# Retries even if the --retry-connrefused is not available
+function common_curl() {
+
+    if [ -n "${connrefused}" ]; then
```

Comment:
This variable has changed to `curl_has_connrefused`, hasn't it?

Reply:
I forgot to save this change.

Done in https://github.com/wazuh/wazuh-packages/pull/2063/commits/e6347badb6ca09124650dca6864aa75b3d984ca2.

Reply:
I forgot to save this change.

Done in https://github.com/wazuh/wazuh-packages/pull/2063/commits/e6347badb6ca09124650dca6864aa75b3d984ca2